addons/sale/static/src/js/product_list/product_list.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <t t-name="sale.ProductList">
-        <h4 class="mt-4 mb-3" t-if="this.props.areProductsOptional" t-out="optionalProductsTitle"/>
+        <span name="sale_product_configurator_list_title" class="d-inline-block mt-4 mb-3 h4" t-if="this.props.areProductsOptional" t-out="optionalProductsTitle"/>
```

Comment:
Nit: same here (even though the line underneath doesn't seem to follow this rule).